### DISCUSSION TITLE
Link {More Info} beside entity type {label} opens up a modal, however dropdown elements are overlapping on modal

### DISCUSSION THREAD

**Discussion by kharabela** [tailwindlabs/tailwindcss#13016] [Help] [answered]
https://codepen.io/Kharbelar/pen/PoLrQwx
 what can I do to fix drop down overlapping on modal, I want them to remain hidden like all other elements

**Comment by wongjn** [comment, marked as answer]
You could consider applying a positive `z-index` to the modal element: https://codepen.io/Wongjn/pen/QWoXrXR?editors=1000

**Reply by kharabela** [reply]
you are a life saver.Thanks for reviewing my code in your spare time
